src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -331,6 +331,18 @@ def select_window(self, locator=None):
             pass
         finally:
             self._window_manager.select(self._current_browser(), locator)
+            
+    def get_log(self, log_type):
+        """Get the log for a given selenium log type
+
+        The `log_type` argument defines which logs to get. Possible values are:
+        `browser`, `driver`, `client` or `server`
+        """
+        available_types = ["browser", "driver", "client" , "server"]
+        if log_type not in  available_types:
+            raise  ValueErrorValueError(log_type + " is not a supported type")
```

Comment:
We should not restrict the value of `log_type`. There is a `log_types` attribute on web driver to fetch valid values that makes a call to the driver server ([doc](http://seleniumhq.github.io/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webdriver.html#selenium.webdriver.remote.webdriver.WebDriver.log_types)). This leaves flexibility for other types of logs.